### ISSUE TITLE
Any stage after first one fails with locked index error

### ISSUE THREAD

**Issue by steelbrain** [smashwilson/merge-conflicts#199]
![screenshot from 2016-01-31 00-49-41](https://cloud.githubusercontent.com/assets/4278113/12700948/9c14c440-c7b4-11e5-89d5-ba5e428e22b8.png)

**Comment by smashwilson** [comment]
This is likely a duplicate of #171.

Is it consistently on the second and later attempts? That's a new one.

**Comment by steelbrain** [comment]
@smashwilson Yes it's consistently on second and later attempts, it works for the first time
